invenio_communities/assets/semantic-ui/js/invenio_communities/settings/profile/DangerZone.js

```diff
@@ -15,49 +15,61 @@ import { RenameCommunitySlugButton } from "./RenameCommunitySlugButton";
 import PropTypes from "prop-types";
 import { DeleteCommunityModal } from "./DeleteCommunityModal";
 
-const DangerZone = ({ community, onError }) => (
-  <Segment className="negative rel-mt-2">
-    <Header as="h2" className="negative">
-      {i18next.t("Danger zone")}
-    </Header>
-    <Grid>
-      <Grid.Column mobile={16} tablet={10} computer={12}>
-        <Header as="h3" size="small">
-          {i18next.t("Change identifier")}
-        </Header>
-        <p>
-          {i18next.t(
-            "Changing your community's unique identifier can have unintended side effects."
-          )}
-        </p>
-      </Grid.Column>
-      <Grid.Column mobile={16} tablet={6} computer={4} floated="right">
-        <RenameCommunitySlugButton community={community} onError={onError} />
-      </Grid.Column>
-      <Grid.Column mobile={16} tablet={10} computer={12} floated="left">
-        <Header as="h3" size="small">
-          {i18next.t("Delete community")}
-        </Header>
-        <p>{i18next.t("Once deleted, it will be gone forever. Please be certain.")}</p>
-      </Grid.Column>
-      <Grid.Column mobile={16} tablet={6} computer={4} floated="right">
-        <DeleteCommunityModal
-          community={community}
-          label={i18next.t("Delete community")}
-          redirectURL="/communities"
-          onDelete={async () => {
-            const client = new CommunityApi();
-            await client.delete(community.id);
-          }}
-        />
-      </Grid.Column>
-    </Grid>
-  </Segment>
-);
+const DangerZone = ({ community, onError, permissions }) =>
+  permissions.can_delete || permissions.can_rename ? (
```

Comment:
minor: could we simplify the ternary operator to simpler if/else ? Ternary operators in js are very hard to read, especially if they are stretched through the whole function